README.md

```diff
@@ -15,11 +15,21 @@
 
 ## Setup
 
-1. Update the `.env` file to point to the sde files necessary
-1. Install the dependencies into a conda environment with arcpy
-   - `pip install -r requirements.txt`
```

Comment:
i think you should differentiate between development setup and otherwise

Reply:
There are no differences between development and production setups for this project, no?

Perhaps you are wanting to just show best practice? Would it be just adding an empty `extra_requires` prop to setup.py? e.g. https://github.com/agrc/forklift/blob/733af000e9e53f55a9de3007458770dd6a27c1fa/setup.py#L45-L53

Reply:
no, i wouldn't pip install the module during active development would I? I would continue to use the .env file. I would get the dependencies from pip install -e or however that works.